akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceSpec.scala

```diff
@@ -153,6 +153,29 @@ class SourceSpec extends AkkaSpec {
       gotten.toSet should ===(Set(0, 1, 2, 3, 4))
       out.expectComplete()
     }
+
```

Comment:
I completely understand that rationale, but if you look at it from the readers' point of view, there's really nothing that verifies that it does work and if so what it does.